src/loaders/bitmapFontParser.js

```diff
@@ -84,6 +85,8 @@ module.exports = function ()
 
             resource.bitmapFont = data;
 
+            text.BitmapText.fonts[data.font] = data;
```

Comment:
This puts a hard dependency of the `text` plugin into `loader` now. If I exclude text from my pixi bundle with a command line param, this code will make it still be bundled anyway.

I think we should do this a different way. Like instead of bitmap text looking some global cache, just pass the loaded data into it.